core/app/modules/import-export/assets/js/shared/process-failed-dialog/process-failed-dialog.js

```diff
@@ -42,17 +43,27 @@ export default function ProcessFailedDialog( { errorType, onApprove, onDismiss,
 			} else {
 				window.open( 'http://go.elementor.com/app-import-download-failed', '_blank' );
 			}
+
+			if ( 'kit-library' === referrer && learnMoreEvent) {
+				learnMoreEvent();
+			}
 		},
 		handleOnDismiss = () => {
 			if ( 'general' === error && onDismiss ) {
 				onDismiss();
-			} else if ( 'kit-library' === referrer ) {
+			} else if ( 'kit-library' === referrer && onModalClose ) {
+				onModalClose();
 				navigate( '/kit-library' );
 			} else {
 				action.backToDashboard();
 			}
 		};
 
+		useEffect( () => {
+			if ( 'kit-library' === referrer && isError ) {
```

Comment:
Why call this callback only on kit library referer?

Reply:
Because the event applies only to kit-library, I don't want the event to fire in non-kit library referral actions.